platforms/core-configuration/file-collections/src/integTest/groovy/org/gradle/api/file/RelativePathFilesIntegrationTest.groovy

```diff
@@ -97,6 +97,7 @@ class RelativePathFilesIntegrationTest extends AbstractIntegrationSpec {
 
     // TODO: write a similar test for the RegularFileProperty
     @ToBeFixedForConfigurationCache
+    @Issue("https://github.com/gradle/gradle/issues/322591")
```

Comment:
@ghale, can these todos be removed with your changes?
Also, there is a typo in the issue link

Reply:
Good catch on the broken link.  I don't think I can remove the TODO because (as I read it) it refers to creating a RegularFileProperty test for #32591.  I didn't addd one for that issue, I just added the `@Issue` annotation on this test to be clear which issue it refers to.  My changes (and the tests I added) relate to a different issue (#32992). 

I've got another PR ready to fix the broken link (#33852).